<launch>

    <arg name="intopic" default="/skeleton_data/incremental"/>
    <arg name="X" default="2"/>
    <arg name="Y" default="3"/>
    <arg name="outtopic" default="/skeleton_data/incremental_reduced"/>

    <arg name="rec_rgb" default="True"/>
    <arg name="max_frames" default="1000"/>
    <arg name="log_to_mongo" default="True"/>
    <arg name="soma_map" default="collect_data_map_cleaned"/>
    <arg name="restrict_to_rois" default="True"/>

  <!-- Drops human pose estimate msgs to be logged - check topic_tools drop-->
  <node pkg="topic_tools" type="drop" name="drop" output="screen"
    args="$(arg intopic) $(arg X) $(arg Y) $(arg outtopic)" >
  </node>

  <!-- Listens to and Logs Skeleton Poses from /skeleton_tracker -->
	<node pkg="activity_data" type="skeleton_publisher.py" name="skeleton_publisher" output="screen">
       <param name="rec_rgb" value="$(arg rec_rgb)" type="bool"/>
       <param name="log_to_mongo" value="$(arg log_to_mongo)" type="bool"/>
       <param name="max_frames" value="$(arg max_frames)" type="int"/>

	</node>

</launch>
